src/main/java/com/waratek/spiracle/file/AbstractFileServlet.java

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
This class contains methods factored out from FileServlet.java, it's not new code.